packages/core/test/unit/query-interface/create-table.test.ts

```diff
@@ -0,0 +1,96 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { DataTypes, JSON_NULL, sql } from '@sequelize/core';
+import { createSequelizeInstance, expectsql, sequelize } from '../../support';
+
+const dialect = sequelize.dialect;
+
+describe('QueryInterface#createTable', () => {
+  afterEach(() => {
+    sinon.restore();
+  });
+
+  it('supports sql.uuidV4 default values', async () => {
```

Comment:
For this and the following test; do we have a policy on testing on earlier (supported) versions?

Reply:
I think we should cover all possible branches, done